README.md

```diff
@@ -1,2 +1,71 @@
 # docker-itchysats
-ItchySats in a Docker container
+
+A repo facilitating running ItchySats inside a [Docker][https://www.docker.com/what-docker] container on a x86-64 (e.g. macOS, Linux, Windows) or ARMv7 system.
+Docker website provides extensive tutorials on Docker installation, please refer there if you are new to Docker.
+
+The following guide assumes that you have Docker running, having [docker-compose][https://docs.docker.com/compose/install/] installed is also recommended.
+
+# Quickstart
+
+## Testnet
+
+If you just want to check out the application, and want to try out a "demo" version, where you don't trade with real money, then this config is for you.
+
+```
+docker-compose project-directory=testnet up -d
+```
+
+## Mainnet
+
+If you're ready to trade with real money, use the following command:
+
+```
+docker-compose project-directory=testnet up -d
+```
+
+## User interface
+
+Docker command starts an ItchySats service running in the background.
+The service can be controlled via the provided web UI accessible on [https://localhost:8000][https://localhost:8000]
+Web UI requires authentication via password generated from your `taker_seed`.
+
+### Credentials
+
+Credentials can be seen in Docker logs after typing in the following command (example for `mainnet`):
+
+```
+docker-compose project-directory=mainnet logs
+```
+
+It is strongly recommended to not store ItchySats logs outside your machine without filtering the credentials out.
+
+## Stopping ItchySats
+
+On the example of mainnet:
+
+```
+docker-compose project-directory=mainnet down
+```
+
+# How to use ItchySats?
+
+You can find more information about ItchySats (along with complete source code) on the main github [repo][https://github.com/itchysats/itchysats].
+[FAQ][https://github.com/itchysats/itchysats/blob/master/FAQ.md] might be another good source of knowledge if you get stuck.
+
+# Caveats
+
+- at this stage there is no automated backup of the wallet seed and the database (please backup up 'data' directory inside `testnet` or `mainnet` and store it securely)
```

Comment:
```suggestion
- at this stage there is no automated backup of the wallet seed and the database (please backup up `taker_seed` inside `testnet` or `mainnet` and store it securely)
```

I think they way you configure `testnet` and `mainnet` to be mounted directly, there is no  `data` directory. Or am I missing something?

Reply:
these is a `data` directory inside mainnet and testnet in this config. You're welcome to try :)